### ISSUE TITLE
Separate repo for Vim colorscheme?

### ISSUE THREAD

**Issue by jan-warchol** [jan-warchol/selenized#66]
Migrating from issue #47:

> [Make selenized theme] available in a separate repository ie like [altercation/vim-colors-solarized](https://github.com/altercation/vim-colors-solarized) the reason for doing this is it can then easily be used by a plugin manager (vimplug etc).

**Comment by ianchanning** [comment]
Yes please - life is so much easier if this is a separate repo. I've just created one here for myself: https://github.com/ianchanning/vim-selenized

I use [vim-plug](https://github.com/junegunn/vim-plug), but any of the plugin managers would work and then it's just adding this into my `.vimrc`:

```vim
Plug 'ianchanning/vim-selenized'
```

Works beautifully.

Otherwise I don't actually have a `/colors` directory, all my colorschemes were installed via plugins.

**Comment by toniz4** [comment]
I second this, having the plugin manager handle the download is much more easier then manually cloning the repo and copying the files to the colors directory.

**Comment by juanibiapina** [comment]
Or you can just use:
```vimscript
Plug 'jan-warchol/selenized', { 'rtp': 'editors/vim' }
```

**Comment by ianchanning** [comment]
> Or you can just use:
> 
> ```viml
> Plug 'jan-warchol/selenized', { 'rtp': 'editors/vim' }
> ```

Good idea - but that only works for Plug. There are lots of other plugin managers.
